### PR TITLE
Set environment variable for loaners

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

**Comment by jonasfj** [comment]
In talks on how to make loaners awesome:
https://public.etherpad-mozilla.org/p/jonasfj-lets-make-loaners-awesome

We concluded that we should set an environment variable on loaner tasks... and let in-tree default command take care of setting up an environment, etc... without running the actual tests...

File: lib/ui/loaner-button.jsx [hunk 1]
```diff
@@ -81,15 +81,18 @@ let LoanerButton = React.createClass({
               this.props.taskId + "\\n" +
               "Original command was: " + shellescape(task.payload.command);
 
+    task.payload.env = task.payload.env || {};
+    task.payload.env.TASKCLUSTER_INTERACTIVE = 'true';
+
     // Construct new task command
-    task.payload.command = [
+    /*task.payload.command = [
```
Comment (imbstack) [inline lib/ui/loaner-button.jsx]: Why comment this out and not just delete it?

Comment (jonasfj) [inline lib/ui/loaner-button.jsx]: good point...

Comment (jonasfj) [inline lib/ui/loaner-button.jsx]: fixed...

**Comment by imbstack** [comment]
r+

**Pull request merged**